policyengine_us/parameters/gov/states/la/tax/income/credits/cdcc/agi_threshold.yaml

```diff
@@ -0,0 +1,16 @@
+description: Louisiana determines the refundability of the CDCC based on this federal AGI threshold.
```

Comment:
```suggestion
description: Louisiana makes its child and dependent care credit refundable if a filer's federal adjusted gross income is equal to or less than this amount; otherwise, they allow the credit to be carried forward for up to five years, in which case PolicyEngine treats it as non-refundable.
```